IdentityCore/src/logger/MSIDLogger.m

```diff
@@ -174,12 +260,14 @@ - (void)logWithLevel:(MSIDLogLevel)level
                 NSLog(@"%@", log);
             }
             
-            if (self.callback)
+            if (self.callback || self.loggerConnector)
             {
                 NSString *log = [NSString stringWithFormat:@"%@ %@ %@ %@ [%@%@]%@%@ %@", threadInfo, sdkName, sdkVersion, [MSIDDeviceId deviceOSId], dateStr, correlationIdStr, componentStr, sourceInfo, message];
                 
                 BOOL lineContainsPII = self.PiiLoggingEnabled ? containsPII : NO;
```

Comment:
should we use instance variable here for PiiLoggingEnabled? Since this block is already synchronized and accessing PIILoggingEnabled will do additional unneeded synchronization.